README.md

```diff
@@ -111,38 +73,344 @@ requirements that this script has that may not already be in place:
+* mscs-server-location - Assign the location of the server .jar file.
+* mscs-server-command - Assign the command to run for the server.
+
+The following variables may be used in some of the values of the above keys:
+* $JAVA - The Java virtual machine.
+* $CURRENT_VERSION - The current Mojang Minecraft release version.
+* $CLIENT_VERSION - The version of the client software.
+* $SERVER_VERSION - The version of the server software.
+* $SERVER_JAR - The .jar file to run for the server.
+* $SERVER_ARGS - The arguments to the server.
+* $INITIAL_MEMORY - The initial amount of memory for the server.
+* $MAXIMUM_MEMORY - The maximum amount of memory for the server.
+* $SERVER_LOCATION - The location of the server .jar file.
 
+#### Example key/value pairs
 
-## Usage
+    mscs-enabled=true
+    mscs-version-type=release
+    mscs-client-version=$CURRENT_VERSION
+    mscs-client-jar=$CLIENT_VERSION.jar
+    mscs-client-url=https://s3.amazonaws.com/Minecraft.Download/versions/$CLIENT_VERSION/$CLIENT_VERSION.jar
+    mscs-client-location=/opt/mscs/client/$CLIENT_VERSION
+    mscs-server-version=$CURRENT_VERSION
+    mscs-server-jar=minecraft_server.$SERVER_VERSION.jar
+    mscs-server-url=https://s3.amazonaws.com/Minecraft.Download/versions/$SERVER_VERSION/minecraft_server.$SERVER_VERSION.jar
+    mscs-server-args=nogui
+    mscs-initial-memory=128M
+    mscs-maximum-memory=2048M
+    mscs-server-location=/opt/mscs/server
+    mscs-server-command=$JAVA -Xms$INITIAL_MEMORY -Xmx$MAXIMUM_MEMORY -jar $SERVER_LOCATION/$SERVER_JAR $SERVER_ARGS
+    
+#### Enabling Forge, BungeeCord, and other server software
+Please visit the [wiki]
+(https://github.com/sandain/MinecraftServerControlScript/wiki/Server-Customization-Examples) 
+for additional information.
+
+### Adjusting global server settings
+In tandem with the `mscs.properties` file--which handles options for individual 
+worlds--
+is the `mscs.conf` or simply `mscs` file, which handles global server settings.
+
+**You must create this file**. By default, MSCS looks for this file in three 
+places (in this order):
+
+1. `$HOME/mscs.conf` 
+
+2. `$HOME/.config/mscs/mscs.conf`
+
+3. `/etc/default/mscs`-- **if you save it in this location it is only called 
+`mscs`, NOT `mscs.conf`**
+
+It doesn't matter where you create the file, as long as you put it in one of 
+the above places.
+
+Please note: `$HOME` represents the home directory of the user that is 
+responsible for the script--
+if you followed the configuration above, then that would be the `minecraft` 
+user. 
+To print the home directory of the `minecraft` user type:
+
+`eval echo ~$minecraft`
+
+Then simply `cd` into the directory it outputs and create the file.
+
+Once you've created the file, you need to populate it with a list of properties. 
+The list of properties can be found 
+[here](https://github.com/sandain/MinecraftServerControlScript/wiki/Global-Server-Settings).
+
+## Automated backups and restarts
```

Comment:
Sounds good, though when you say create another section with mapping and other available commands, what do you mean? B/c I did make a mapping section & there is a command list section.